cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,58 @@
 /// <reference types='cypress' />
 
-describe('Book Store app', () => {
-  before(() => {
+describe('BookStore app', () => {
+  const user = {
+    userName: 'Superqa007',
+    password: 'Supertestqa1!'
+  };
+
+  const book = {
+    title: 'Speaking JavaScript',
+    author: 'Axel Reuschmayer',
+    publisher: 'D\'Reilly Media',
+    description: 'Like it or not, JavaScript is everywhere these days'
+  };
 
+  const alertMessage = {
+    added: 'Book added to your collection.',
+    deleted: 'Book deleted.'
+  };
+
+  before(() => {
+    cy.visit('/login');
   });
 
-  it('', () => {
+  it('should allow to login', () => {
```

Comment:
I think test should be named differently - you're checking not only login)

Reply:
Agree